catalog/MDCCatalog/MDCCatalogTiles.m

```diff
@@ -3044,10 +3044,10 @@ void MDCCatalogDrawPageControlTile(CGRect frame, NSObject<MDCColorScheme> *color
   CGColorSpaceRelease(colorSpace);
 }
 
-static NSString* const PalettesFontName = @"RobotoMono-Regular";
-
 void MDCCatalogDrawPalettesTile(CGRect frame, NSObject<MDCColorScheme> *colorScheme) {
   CGFloat PalettesFontPointSize = [UIScreen mainScreen].bounds.size.width / 320 * 7;
+   
+   static NSString* const PalettesFontName = @"RobotoMono-Regular";
```

Comment:
Please revert the formatting changes to this file.
(Cleanup is welcome, but we like to keep them in their own PRs)